aquifers/AQUFLUX-02.DATA

```diff
@@ -0,0 +1,307 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+-- Copyright (C) 2023 SINTEF ICT, Applied Mathematics
+
+-- Test case is testing analytical aquifer - constant flux
```

Comment:
I think we should expand a little bit here to note how `-02` differs from `-01`.